runner/components/io.siddhi.distribution.metrics.core/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--<Configuration>-->
```

Comment:
Remove

Reply:
The above comment is addressed by the commit 80ebd14.